apps/src/code-studio/components/progress/teacherPanel/StudentTable.jsx

```diff
@@ -96,8 +95,7 @@ class StudentTable extends React.Component {
                   )}
                   <div style={styles.name}>
                     {student.name}
-                    {!!DCDO.get('family-name-features', false) &&
```

Comment:
A reminder for anyone who isn't familiar with how this functionality works- the format of these calls is .get('flag-name', defaultValue).